code/datums/elements/consumable_mob.dm

```diff
@@ -0,0 +1,32 @@
+/**
+ * element for mobs that can be consumed!
+ */
+/datum/element/consumable_mob
```

Comment:
ideally this would have a "max eats" variable where after you eat the thing enough times it calls `death()` but whatever for right now (no change)